src/components/element-editor/validation/validation-child.tsx

```diff
@@ -0,0 +1,96 @@
+import { Form } from "@lifesg/react-design-system";
+import { useEffect, useState } from "react";
+import { ChildEntry } from "src/components/common";
+import { IValidation } from "src/context-providers/builder";
+import { FieldWrapper } from "./validation.styles";
+
+interface IProps {
+    onDelete: () => void;
+    options: string[];
+    onChange: (newValue: any) => void;
+    value?: IValidation;
+}
+
+export const ValidationChild = ({
+    onDelete,
+    options,
+    onChange,
+    value,
+}: IProps) => {
+    // =========================================================================
+    // CONST, STATE, REF
+    // =========================================================================
+    const [type, setType] = useState<string>();
+
+    // =============================================================================
+    // EVENT HANLDERS
+    // =============================================================================
+    const handleChange = (changeType?: string, newValue?: string) => {
+        const updatedValue = { ...value };
+        if (!updatedValue.validationType || options.length === 1) {
+            updatedValue.validationType = type;
+        }
+        if (changeType === "rule") {
+            updatedValue.validationRule = newValue;
+        } else if (changeType === "errorMessage") {
+            updatedValue.validationErrorMessage = newValue;
+        }
+
+        onChange(updatedValue);
+    };
+
+    // =========================================================================
+    // USE EFFECTS
+    // =========================================================================
+
+    useEffect(() => {
+        if (options.length === 1) {
+            setType(options[0]);
+        } else {
+            setType(value.validationType);
+        }
+    }, [options]);
```

Comment:
- why not call `handleChange` to pass the validationType to parent?
- should default to 1st option if there is only 1 option AND `value.validationType` is not set
- else block is probably not needed if we are doing away with storing type in local state